### ISSUE TITLE
Make the DevOps virtual machines accessible from office wired net

### ISSUE THREAD

**Issue by kr15h** [FabLabBerlin/fablab-networking#2]
Right now the newly installed virtual machine is accessible from Lounge and wired network, but it would be actually more important to configure access from Fab Lab Office and Fab Lab R&D center.

**Comment by kr15h** [comment]
I actually managed to ping the machine 10.42.185.96 only from the Fab Lab Launge network.

**Comment by DatanoiseTV** [comment]
Thats correct. Its on my milestone list and will be fixed as soon as OB sets up the network.
